setup/package.dffedora

```diff
@@ -17,6 +17,7 @@ RUN dnf update -d 0 -e 0 -y && \
         python3-babel \
         python3-dateutil \
         python3-decorator \
+        python3-devel \
```

Comment:
Why do we need devel ??

Reply:
It's [needed](https://docs.fedoraproject.org/en-US/packaging-guidelines/Python/#_buildrequire_python3_devel) for the packaging python stuffs in the Fedora Docker image.